subsys/bluetooth/mesh/shell.c

```diff
@@ -1932,6 +2350,128 @@ static int cmd_provision_adv(const struct shell *shell, size_t argc,
 
 	return 0;
 }
+
+static int cmd_prov_close_link(const struct shell *shell, size_t argc,
+			     char *argv[])
+{
+	pb_adv.link_close(PROV_BEARER_LINK_STATUS_FAIL);
+
+	return 0;
+}
+
+static int cmd_prov_static_oob(const struct shell *shell, size_t argc,
+			     char *argv[])
+{
+	size_t len;
+	int err;
+
+	if (argc < 2) {
+		return -EINVAL;
+	} else {
+		uint8_t static_key[16];
+
+		len = hex2bin(argv[1], strlen(argv[1]),
+					      static_key, 16);
+
+		err = bt_mesh_auth_method_set_static(static_key, len);
+	}
+
+	if (err) {
+		shell_error(shell, "Static OOB value set Failed (err %d)", err);
+	} else {
+		shell_print(shell, "Static OOB value set (size %u)", len);
+	}
+
+	return 0;
+}
+
+static int cmd_prov_input_oob(const struct shell *shell, size_t argc,
+			     char *argv[])
+{
+	int err;
+	size_t size;
+
+	if (argc < 3) {
+		return -EINVAL;
+	} else {
+		bt_mesh_input_action_t act;
+
+		act = (bt_mesh_input_action_t)str2u8(argv[1]);
+		size = str2u8(argv[2]);
+
+		err = bt_mesh_auth_method_set_input(act, size);
+	}
+
+	if (err) {
+		shell_error(shell, "Input OOB Set Failed (err %d)", err);
+	} else {
+		shell_print(shell, "Input OOB value set (size %u)", size);
+	}
+
+	return 0;
+}
+
+static int cmd_prov_output_oob(const struct shell *shell, size_t argc,
+			     char *argv[])
+{
+	int err;
+	size_t size;
+
+	if (argc < 3) {
+		return -EINVAL;
+	} else {
+		bt_mesh_output_action_t act;
+
+		act = (bt_mesh_output_action_t)str2u8(argv[1]);
+		size = str2u8(argv[2]);
+
+		err = bt_mesh_auth_method_set_output(act, size);
+	}
```

Comment:
Remove the `else` in these `return ... else` statements, so we can keep the `else`-block at the innermost indentation